plugins/provenance/server/resource.py

```diff
@@ -329,7 +329,9 @@ def fileSaveHandler(self, event):
                                        level=AccessType.READ, force=True)
         if not item:
             return
-        prevFile = self.model('file').load(curFile['_id'])
+        # should all of these sort of events be forced, since a save has to be
+        # granted first?
+        prevFile = self.model('file').load(curFile['_id'], force=True)
```

Comment:
Sure.

In regards to the code a few lines above, shouldn't ```Item``` model loads that have ```user``` AND ```force``` be retained due to this call:
https://github.com/girder/girder/blob/acl-resolver-for-non-acl-models/girder/models/item.py#L130
which eventually trickles up to
https://github.com/girder/girder/blob/acl-resolver-for-non-acl-models/girder/models/model_base.py#L486

In which case only ```level``` is extraneous if ```force=True```.